packages/client-core/src/user/components/Oauth/DiscordCallback.tsx

```diff
@@ -25,7 +25,7 @@ const DiscordCallbackComponent = (props): any => {
     if (!error) {
       if (type === 'connection') {
         const user = useAuthState().user
-        AuthService.refreshConnections(user.id.value)
+        AuthService.refreshConnections(user.id.value!)
```

Comment:
`user?.id.value && AuthService.refreshConnections(user.id.value)`

do the above if there is no point in calling the service without the user id value.

Reply:
Please replace it everywhere in PR as well if true.